src/main/resources/hudson/plugins/active_directory/Messages.properties

```
@@ -13,4 +13,5 @@ TlsConfiguration.TrustAllCertificates = (Insecure) Trust all Certificates
 TlsConfiguration.JdkTrustStore = JDK TrustStore
 
 TlsConfiguration.AdministrativeMonitor.DisplayName = Active Directory TLS Configuration Monitor
-TlsConfiguration.ErrorMessage = Disabling TLS in FIPS mode is not allowed. Either enable StartTls or Require TLS.
\ No newline at end of file
+TlsConfiguration.ErrorMessage = Disabling TLS in FIPS mode is not allowed. Either enable StartTls or Require TLS.
+TlsConfiguration.CertificateError = Choosing an insecure certificate in FIPS mode is not allowed. Please choose a secure certificate.
```

Trusting all certificates is not an insecure certificate, but an insecure TLS configuration
```suggestion
TlsConfiguration.CertificateError = Choosing an insecure TLS configuration in FIPS mode is not allowed. Please do not trust all certificates.
```